samples/net/wifi/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _wifi:
+
+Wi-Fi sample
+############
+
+Overview
+********
+
+This sample enables testing Wi-Fi drivers for various boards.
+It simply enables the Wi-Fi shell module that will provide a set
+of commands: scan, connect and disconnect.
+It also enables net shell module to verify net if settings.
+
+Building and Running
+********************
+
+Make sure the board and the chip you are targeting are supported for
```

Comment:
How do you make sure of this?  Should this read:

    Verify the board and chip you are targeting provide Wi-Fi support.